Sources/ReplyService/ReplyRequest.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  SendMessageRequest.swift
+//  SwiftBot
+//
+//  Created by Andrew Tokarev on 15/06/2017.
+//
+//
+
+import Foundation
+
+public protocol ReplyRequest
```

Comment:
May be we can merge ReplyRequest and FacebookGraph (GraphQL), what do you think?